full/build.gradle

```diff
@@ -45,65 +44,14 @@ shadowJar {
-//    implementation group: 'commons-codec', name: 'commons-codec', version: '1.14'
-    implementation group: 'com.jayway.jsonpath', name: 'json-path', version: '2.4.0'
-    implementation group: 'org.hdrhistogram', name: 'HdrHistogram', version: '2.1.9'
-    compileOnly group: 'org.neo4j.driver', name: 'neo4j-java-driver-slim', version: '4.4.5'
-    implementation group: 'com.novell.ldap', name: 'jldap', version: '2009-10-07'
 
     antlr "org.antlr:antlr4:4.7.2", {
         exclude group: 'org.glassfish'
         exclude group: 'com.ibm.icu'
         exclude group: 'org.abego.treelayout'
     }
-    // compileOnly "org.antlr:antlr4-runtime:4.7.2"
-    // testImplementation "org.antlr:antlr4-runtime:4.7.2"
-
-    compileOnly "org.apache.poi:poi:5.1.0"
-    testImplementation "org.apache.poi:poi:5.1.0"
-    compileOnly "org.apache.poi:poi-ooxml:5.1.0"
-    testImplementation "org.apache.poi:poi-ooxml:5.1.0"
-
-    implementation 'org.jsoup:jsoup:1.14.3'
-
-    // currently we cannot update selenium to the latest version due to guava minimum version required (31.0.1-jre)
-    compileOnly group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.141.59', { exclude group: 'com.google.guava', module: 'guava' }
-    testImplementation group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.141.59', { exclude group: 'com.google.guava', module: 'guava' }
-    compileOnly group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '5.1.0'
-    testImplementation group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '5.1.0'
-    
-    implementation group: 'org.roaringbitmap', name: 'RoaringBitmap', version: '0.7.17'
-    implementation(group: 'org.apache.commons', name: 'commons-configuration2', version: '2.8.0') {
-        exclude group: "org.yaml"
-        exclude module: "snakeyaml"
-        exclude module: "commons-lang3"
-    }
-    implementation group: 'org.yaml', name: 'snakeyaml', version: '1.26'
-    testImplementation group: 'com.github.stefanbirkner', name: 'system-rules', version: '1.19.0'
-
-    testImplementation 'net.sourceforge.jexcelapi:jxl:2.6.12'
-
-    compileOnly group: 'com.amazonaws', name: 'aws-java-sdk-s3', version: '1.11.270'
-    testImplementation group: 'com.amazonaws', name: 'aws-java-sdk-s3', version: '1.11.270'
-
-    testImplementation group: 'org.reflections', name: 'reflections', version: '0.9.12'
-
-    testImplementation group: 'org.hamcrest', name: 'hamcrest-library', version: '1.3'
-    testImplementation group: 'org.neo4j.test', name: 'neo4j-harness', version: neo4jVersionEffective
-    testImplementation group: 'org.eclipse.jetty', name: 'jetty-server', version: '9.2.22.v20170606'
-    testImplementation group: 'org.apache.derby', name: 'derby', version: '10.12.1.1'
-
-    testImplementation group: 'org.postgresql', name: 'postgresql', version: '42.1.4'
```

Comment:
It looks like a test for the Postgres driver is failing for a genuine reason. Could be that this Postgres dependency is needed?

You may wish to run the tests locally if you haven't already, since many of these types of tests are probably disabled in the CI.

Reply:
Yep good catch, I've brought back this, but I don't understand how the code compiles if we need it in runtime